src/main/webapp/teacher/executionCourse/attendsSearch/attendsSearchApp.js

```diff
@@ -1,19 +1,35 @@
-
+//Copied from http://stackoverflow.com/questions/1584370/how-to-merge-two-arrays-in-javascript-and-de-duplicate-items
+Array.prototype.unique = function() {
+    var a = this.concat();
+    for(var i=0; i<a.length; ++i) {
+        for(var j=i+1; j<a.length; ++j) {
+            if(a[i].id === a[j].id)
+                a.splice(j--, 1);
+        }
+    }
+    return a;
+};
+//END OF COPY
```

Comment:
why not use [$.unique](https://api.jquery.com/jQuery.unique/) or using [vanilla](http://stackoverflow.com/a/14438954) javascript ?